controller:
  ros__parameters:
    waypoints_x: [0.0] # [m] DO NOT MODIFY IT
    waypoints_y: [0.0] # [m] DO NOT MODIFY IT
    distance_margin: 0.50 # [m] DO NOT MODIFY IT
    Kp: [0.2, 1.2] # Your parameter here, proportional gains [linear, angular]
    Kd: [0.01, 0.05] # Your parameter here, derivative gains [linear, angular]
    Ki: [0.0, 0.0] # Your parameter here, integral gains [linear, angular]
